Mencoba code review

kelvinpython123-ai opened the pull request:
Saya Kelvin Python sebagai kontributor dari repositor @kelvinbuh di belajar-kolaborasi ingin menambahkan beberapa kalimat di README.md. Saya harap anda dapat memeriksanya terlebih dahulu.

kelvinpython123-ai committed:
Mencoba code review

Latihan code review

kelvinbuh commented on README.md:
```diff
@@ -1,2 +1,3 @@
 # belajar-kolaborasi
-Perubahan ke-3
+Belajar Github di Dicoding sangat menyenangkan.
```
Perlu menambahkan tag <<**br**>> karena baris ke empat menjadi satu dengan baris ke tiga.

kelvinpython123-ai committed:
Revisi README.md (Latihan Code Review)

Merevisi pada line 2 memasukan <<**br**>> agar line 2 dan line 3 memiliki jarak spasi.

kelvinpython123-ai reviewed (commented):
Hai, perubahan sudah saya lakukan. Silahkan diperiksa kembali.

kelvinbuh commented:
Terima kasih telah berkontribusi dalam repositori ini ya @kelvinpython123-ai
